Malt/Pipelines/NPR_Pipeline/Shaders/NPR_Pipeline/NPR_Mesh.glsl

```diff
@@ -152,5 +152,15 @@ bool is_main_pass()
     }
     #endif
 }
+/*META @meta: category=Input; */
+void pass_info(
+    out bool main_pass,
```

Comment:
For bool properties I would like to use the *Is* naming, so *Shadow Pass* -> *Is Shadow Pass*.

Reply:
having the out variable have the same name as a function doesnt work apparently. Is there a way around it? or would we just make META labels for those outputs?

Reply:
You can name the out variable Is_Main_Pass, or simply use a label.

Reply:
How do you add a label to an output? Can you even do that? Because generall only the inputs/parameters can have meta data right? Because of that I went with the first option